transport/src/commonMain/kotlin/com/genesys/cloud/messenger/transport/core/events/Event.kt

```diff
@@ -32,4 +32,9 @@ sealed class Event {
         val message: String?,
         val correctiveAction: CorrectiveAction,
     ) : Event()
+
+    /**
+     * This event indicates that conversation was successfully autostart.
```

Comment:
perhaps "autostarted"?

Reply:
Does it feel correct to rename the event to "ConversationAutostarted" as well?

Reply:
I'm iffy on that one. It's better _English_ but I don't know if it makes sense to change it. @crumpf what say you?

Reply:
> I'm iffy on that one. It's better _English_ but I don't know if it makes sense to change it. @crumpf what say you?

I'm fine with it staying `ConversationAutostart` instead of using the -ed postfix.